daemon/entrypoint.sh

```diff
@@ -276,7 +276,12 @@ function osd_directory {
   fi
 
   for OSD_ID in $(ls /var/lib/ceph/osd |  awk 'BEGIN { FS = "-" } ; { print $2 }'); do
-    if [ -n "${JOURNAL_DIR}" ]; then
+    OSD_JOURNAL_TMP="OSD_JOURNAL$OSD_ID"
+    OSD_JOURNAL=${!OSD_JOURNAL_TMP}
+    if [ -n "${OSD_JOURNAL}" ]; then
+      ceph-disk -v prepare ${OSD_JOURNAL}
```

Comment:
My intention is to prepare a journal for the corresponding OSD-id. So for example, when I have the dir tree as: /var/lib/ceph/osd/ceph-13, /var/lib/ceph/osd/ceph-25; and assuming I have created two partitions to use as journals: /dev/sdb1, /dev/sdb2; then I could pass `-e OSD_JOURNAL13=/dev/sdb1 -e OSD_JOURNAL25=/dev/sdb2` in the `docker run` command and expect that sdb1 will be prepared and made as journal for OSD.13 and sdb2 for OSD.25.